cmd/java/functions_framework/main.go

```diff
@@ -178,6 +178,12 @@ func mavenClasspath(ctx *gcp.Context) (string, error) {
 // because apparently you can't define tasks there; and having the predefined script include the user's build.gradle
 // didn't work very well either, because you can't use a plugins {} clause in an included script.
 func gradleClasspath(ctx *gcp.Context) (string, error) {
+
```

Comment:
No newline here.

Reply:
removed but `mavenClasspath` has this new line =)